packages/core/src/hooks/useContractCall.ts

```diff
@@ -9,7 +9,10 @@ export function useContractCall(
   args: any[] | Falsy
 ): any | undefined {
   const callData = args && args.every((arg) => arg !== undefined) && abi.encodeFunctionData(functionName, args)
-
   const result = useChainCall(callData && address && { address, data: callData })
+  if (result === '0x') {
+    console.warn(`Invalid contract call: address=${address} function=${functionName} args=${args}`)
+    return undefined
+  }
```

Comment:
Hmmm, do we wanna do console.logs? @JustynaBroniszewska 
Perhaps throw exception? Not sure, cause it will break all the calls.

Reply:
Current multicall contract implementation doesnt allow us to distinguish at which call the error happened. Until we deploy our own implementation we cant really do proper error handling.

So our options here are:
a) Print error and return undefined
b) Crash the whole app

Considering that errors in view functions should be extremely rare and usually indicate a developer error I thing a) is a better option.